SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -0,0 +1,51 @@
+# Get started: 2 - Get and build the GDK’s Unreal Engine Fork
+
+To use the Unreal GDK, you need to build our SpatialOS fork of Unreal Engine 4 from source.
```

Comment:
Should we one-sentence justify it here? "As the GDK extends Unreal Engine's networking capabilities at its core..."